test/unit/lib/index_test.js

```diff
@@ -232,6 +231,17 @@ describe('Tagged API', function() {
             this.http.verifyNoPendingRequests();
             return promise.should.be.rejected;
         });
+
+        it('resolves promise if response bdy is null', function() {
+            var expectedResult = {stat: 'ok'};
```

Comment:
In this case, I think the expected result should be:

    { stat: "ok", result: null }